voice/bxml/callbacks/recordComplete.md

```diff
@@ -59,6 +60,7 @@ POST http://[External server URL]
 	"recordingId"   : "r-115da407-e3d9-4ea7-889f-5f4ad7386a80",
 	"callUrl"       : "https://../{accountId}/calls/{callId-1}",
 	"channels"      : 1,
+	"answerTime"    : "2019-09-13T16:48:26.665Z",
 	"startTime"     : "2019-09-13T16:48:29.235Z",
```

Comment:
`answerTime` should not be earlier than `startTime`.

Reply:
In this callback, `startTime` is the recording start time. Trick one!